src/content/overlay.ts

```diff
@@ -163,17 +168,38 @@ export default class Overlay {
     // If we have a number that we can click
     if (pathNumber - 1 < this.clickables.length) {
       const node = this.clickables[pathNumber - 1];
-      if ((node as HTMLElement).className.includes("CodeMirror")) {
-        (node as HTMLElement).getElementsByTagName("textarea")[0].focus();
-        (node as HTMLElement).getElementsByTagName("textarea")[0].click();
-      } else {
-        (node as HTMLElement).focus();
-        (node as HTMLElement).click();
-      }
+      this.clickNode(node);
     }
     this.clearOverlays();
   }
 
+  public domClick(query: string) {
+    const node = document.querySelector(query);
+    if (node !== null) {
+      this.clickNode(node);
+    } else {
+      return;
+    }
```

Comment:
I think the `else` here and below doesn't do anything, since the function is going to return anyway?